src/pages/blog/airship-update-february-2021.md

```diff
@@ -0,0 +1,120 @@
+<!-- more -->
+
+## **AIRSHIP BLOG**
+
+The Airship blog is a great way to keep up with what's going on in the community. The Airship community publishes
+[_blog posts_](https://www.airshipit.org/blog/) regularly, including the Airship 2.0 Blog series. These blog posts
+introduce the changes from Airship 1.0 to Airship 2.0, highlight new features, and detail each component's evolution.
+The recommended reading order for these blogs is listed below. It begins with the lessons learned on the road to Alpha
+to address the changes made from the original blog series 1-6.
+
+- [*Airship 2.0 is Alpha - Lessons Learned*](https://www.airshipit.org/blog/airship2-is-alpha/)
+- [*Pre-Alpha Airship Blog Series 1 - Evolution Towards 2.0*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-1-evolution-towards-2.0/)
+- [*Pre-Alpha Airship Blog Series 2 - An Educated Evolution*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-2-an-educated-evolution/)
+- [*Pre-Alpha Airship Blog Series 3 - Airship 2.0 Architecture High Level*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-3-airship-2.0-architecture-high-level/)
+- [*Pre-Alpha Airship Blog Series 4 - Shipyard - an Evolution of the Front Door*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-4-shipyard-an-evolution-of-the-front-door/)
+- [*Pre-Alpha Airship Blog Series 5 - Drydock and Its Relationship to Cluster API*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-5-drydock-and-its-relationship-to-cluster-api/)
+- [*Pre-Alpha Airship Blog Series 6 - Armada Growing Pains*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-6-armada-growing-pains/)
+
+<br>
+
+Stay tuned for upcoming blog posts as Airship 2.0 progresses through release!
+
+<br>
+
+## **AIRSHIP 2020 ANNUAL REPORT**
+
+Each year the Open Infrastructure Foundation (OIF) provides an annual report to the community, including project
+developments, foundation news, and letters from the Executive Director. You can find the 2020 OIF report [_here_](
+https://www.openstack.org/annual-reports/2020-openinfra-foundation-annual-report/).
+
+The Airship project began issuing its annual reports, included in the yearly OIF report, at the end of 2019. That
+report is available [_here_](https://www.airshipit.org/blog/a-year-in-review-getting-confirmed-and-looking-ahead-to-2-0/).
+
+Last month Drew Walters authored the [_Airship 2020 Annual Report_](https://www.airshipit.org/blog/airship-2020-annual-report/).
+I encourage you to read the full report; a few of the highlights you'll find include:
+
+* Airship 2.0 GA Status
+* Airship designated as a Certified Kubernetes Distribution
+* Community metrics
+* Cloud Infrastructure Telco Taskforce (CNTT) updates
+
+<br>
+
+## **COMMUNITY MEETING AGENDAS**
+
+Since the Airship IRC weekly meeting changed from a weekly schedule to a biweekly meeting schedule, we thought it would
```

Comment:
I'd suggest changing "we" from a pronoun to a proper noun, i.e. "The Working Committee"

Reply:
Done